File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.TrainingService.Models/remove-training-event-participants_param.ts


import { IRemoveTrainingEventParticipants_Param } from './iremove-training-event-participants_param';

export class RemoveTrainingEventParticipants_Param implements IRemoveTrainingEventParticipants_Param {
  
	public TrainingEventID: number = 0;
	public PersonIDs: number[] = null;
	public RemovalReasonID?: number;
	public RemovalCauseID?: number;
	public DateCanceled?: Date;
  
}

File: GTTS/INL.TrainingService.Models/RemoveTrainingEventParticipants_Param.cs
using System;

namespace INL.TrainingService.Models
{
    public class RemoveTrainingEventParticipants_Param : IRemoveTrainingEventParticipants_Param
    {
        public long TrainingEventID { get; set; }
        public long[] PersonIDs { get; set; }
        public long? RemovalReasonID { get; set; }
        public long? RemovalCauseID { get; set; }
        public DateTime? DateCanceled { get; set; }
    }
}
